### ISSUE TITLE
XML file error in MechanicTranslationalLosslessConnector.xml

### ISSUE THREAD

**Issue by IngoSt** [Hopsan/hopsan#2134]
Failure loading the standard library due to an error in the 
MechanicTranslationalLosslessConnector.xml
Resolve:

add `</help>` in (new) line 11 in the componentLibraries\defaultLibrary\Mechanic\Linear\MechanicTranslationalLosslessConnector.xml file

**Comment by Tirpitz93** [comment]
I believe this has been in fixed in #53e9c48cb8753caa0018d5c3857925ca1b6b31bd 
But no release has been made since

**Comment by peterNordin** [comment]
Yes, sorry I was in the progress of making a release, but then we tried to introduce FMI3 support and its not entirely finished.
Then I got sidetracked, I will try to build the release I initially intended today.

**Comment by peterNordin** [comment]
The new release has now been uploaded to github

**Comment by peterNordin** [comment]
@IngoSt what was the error? Was it possible to continue using Hopsan even tough you got that error ?
